site/content/blogs/release-v121.ja.md

```diff
@@ -0,0 +1,157 @@
+---
+title: 'AWS Copilot v1.21: CloudFront が登場!'
+twitter_title: 'AWS Copilot v1.21'
+image: 'https://user-images.githubusercontent.com/10566468/184949047-f4f173ae-0b29-47fd-8c0b-14a212029587.png'
+image_alt: 'v1.21 Environment Manifest と Lambda アドオン'
+image_width: '1051'
+image_height: '747'
+---
+
+# AWS Copilot v1.21: CloudFront が登場!
+
+投稿日: 2022 年 8 月 17 日
+
+AWS Copilot コアチームは、Copilot v1.21 リリースを発表します。
+このリリースに貢献してくれた [@dave-moser](https://github.com/dave-moser)、[@dclark](https://github.com/dclark)、[@apopa57](https://github.com/apopa57) に特別な感謝を捧げます。私たちのパブリックな[コミュニティチャット](https://gitter.im/aws/copilot-cli)は成長しており、オンラインでは 300 人以上、[GitHub](http://github.com/aws/copilot-cli/) では 2.4k 以上のスターを獲得しています。AWS Copilot へご支援、ご支持いただいている皆様お一人お一人に感謝をいたします。
+
+Copilot v1.21 では、いくつかの新機能と改良が施されています。
+
+- **CloudFront を Application Load Balancer に統合**: CloudFront を Load Balanced Web Service の手前にデプロイできるようになりました。[詳細はこちらをご覧ください](#cloudfront-%E3%81%AE%E7%B5%B1%E5%90%88)。
+- **Environment セキュリティグループの設定**: Environment Manifest にて Environment セキュリティグループのルールを設定します。[詳しくはこちらをご覧ください](#environment-%E3%82%BB%E3%82%AD%E3%83%A5%E3%83%AA%E3%83%86%E3%82%A3%E3%82%B0%E3%83%AB%E3%83%BC%E3%83%97%E3%81%AE%E8%A8%AD%E5%AE%9A)。
+- **ELB アクセスログのサポート**: Load Balanced Web Service に対して、Elastic Load Balancing のアクセスログを有効にします。[詳しくはこちらをご覧ください](#elb-%E3%82%A2%E3%82%AF%E3%82%BB%E3%82%B9%E3%83%AD%E3%82%B0%E3%81%AE%E3%82%B5%E3%83%9D%E3%83%BC%E3%83%88)。
+- **`job logs` の改善**: Job のログをたどり、ステートマシンの実行ログを見ることができるようになりました。[詳しくはこちらをご覧ください](./#)。
+- **デプロイ前の CloudFormation テンプレートのパッケージ化 Addon**: Copilot は、`copilot svc deploy` コマンドで Addon テンプレートをパッケージ化するようになりました。これにより、Copilot はコンテナ化された Service と一緒に AWS Lambda 関数をデプロイできるようになりました! 導入方法については [Copilotのドキュメント](../docs/developing/addons/package.ja.md) をご覧ください。
+
+???+ note "AWS Copilot とは?"
+
+    AWS Copilot CLI は AWS 上でプロダクションレディなコンテナ化されたアプリケーションのビルド、リリース、そして運用のためのツールです。
+    開発のスタートからステージング環境へのプッシュ、本番環境へのリリースまで、Copilot はアプリケーション開発ライフサイクル全体の管理を容易にします。
+    Copilot の基礎となるのは、 AWS CloudFormation です。CloudFormation により、インフラストラクチャを 1 回の操作でコードとしてプロビジョニングできます。
+    Copilot は、さまざまなタイプのマイクロサービスの作成と運用の為に、事前定義された CloudFormation テンプレートと、ユーザーフレンドリーなワークフローを提供します。デプロイメントスクリプトを記述する代わりに、アプリケーションの開発に集中できます。
+
+    より詳細な AWS Copilot の紹介については、[Overview](../docs/concepts/overview.ja.md) を確認してください。
+
+## CloudFront の統合
+
+Copilot の Environment Manifest への最初の大きな追加の 1 つです! CloudFront は AWS Content Delivery Network (CDN) で、世界中にアプリケーションをディストリビューションするのに役立ちます。Environment Manifest に `cdn: true` を設定して `copilot env deploy` を実行するだけで、ディストリビューションを有効にできるようになりました。
```

Comment:
In your opinion, should only this part be changed from "ディストリビューション" to "配信" and the "ディストリビューション" after this should remain the same? I think just the part you point out is fine, since even on the CloudFront documentation there is a notation "CloudFront ディストリビューションを作成" or "ディストリビューションを使用".